### PR TITLE
add bloomfilter index doc

### PR DESCRIPTION
**Pull request by hf200012**
add bloomfilter index doc

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
How about move the doc to the place where bitmap index locate:
administrator-guide/alter-table/alter-table-bitmap-index.md

File: docs/en/administrator-guide/bloomfilter.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+-A highly space-efficient probabilistic data structure used to check whether an element is in a set.
+-For a call to detect whether an element exists, BloomFilter will tell the caller one of two results: it may exist or it must not exist.
+-The disadvantage is that there is a misjudgment, telling you that it may exist, not necessarily true.
+
+Bloom filter is actually composed of an extremely long binary bit array and a series of hash functions. The binary bit array is all 0 initially. When an element to be queried is given, this element will be calculated by a series of hash functions to map out a series of values, and all values are treated as 1 in the offset of the bit array.
+
+Figure 3-16 below shows an example of Bloom Filter with m=18, k=3 (m is the size of the Bit array, and k is the number of Hash functions). The three elements of x, y, and z in the set are hashed into the bit array through three different hash functions. When querying the element w, after calculating by the Hash function, because one bit is 0, w is not in the set.
```
Comment (whutpencil) [inline docs/en/administrator-guide/bloomfilter.md:37]: Should we remove the serial number of the picture?

Comment (whutpencil) [inline docs/en/administrator-guide/bloomfilter.md:37]: Should we remove the serial number of the picture?

**Comment by hf200012** [comment]
> How about move the doc to the place where bitmap index locate: administrator-guide/alter-table/alter-table-bitmap-index.md

No, I think that the index should be taken out of a directory separately, and the prefix index, Bitmap index and BloomFilter index should be put together

File: docs/en/administrator-guide/bloomfilter.md (hunk 1 above)
Comment (hf200012) [reply docs/en/administrator-guide/bloomfilter.md:37]: remove pic??

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**